protected/humhub/modules/user/models/Password.php

```diff
@@ -74,7 +74,7 @@ public function rules()
     {
         return [
             [['newPassword', 'newPasswordConfirm'], 'required', 'on' => 'registration'],
-            [['newPassword', 'newPasswordConfirm'], 'trim'],
+            [['newPassword', 'newPasswordConfirm'], 'string', 'min' => 5, 'max' => 255],
```

Comment:
Row could be deleted, 

And
`  [['newPassword', 'newPasswordConfirm', 'currentPassword'], 'required', 'on' => 'changePassword'],`
to
`[['newPassword', 'newPasswordConfirm', 'currentPassword'], 'required', 'on' =>['changePassword','registration']],`